test/Tests/src/System/File_Spec.enso

```diff
@@ -120,14 +120,89 @@ spec =
             contents_2.should .start_with "Cupcake ipsum dolor sit amet."
 
     Test.group "write operations" <|
+        data = [32, 127, -128, 0]
+        data_2 = [10, 15, 20, 30]
+
         transient = enso_project.data / "transient"
-        Test.specify "should allow to append to files" <|
+        Test.specify "should allow to writing bytes to a new file" <|
+            f = transient / "new_file.dat"
+            f.delete_if_exists
+            f.exists.should_be_false
+            data.write_bytes f
+            f.exists.should_be_true
+            f.read_bytes.should_equal data
+            f.delete_if_exists
+
+        Test.specify "should backup a file when overwriting with new bytes" <|
+            f = transient / "work.txt"
+            f.delete_if_exists
+            f_bak = transient / "work.txt.bak"
+            f_bak.delete_if_exists
+            data.write_bytes f
+            f.exists.should_be_true
+            data_2.write_bytes f
+            f.read_bytes.should_equal data_2
+            f_bak.exists.should_be_true
+            f.delete_if_exists
+            f_bak.delete_if_exists
+
+        Test.specify "should allow to overwriting a file with new bytes" <|
             f = transient / "work.txt"
             f.delete_if_exists
+            f_bak = transient / "work.txt.bak"
+            f_bak.delete_if_exists
+            data.write_bytes f
+            f.exists.should_be_true
+            data_2.write_bytes f on_existing_file=Existing_File_Behavior.Overwrite
+            f.read_bytes.should_equal data_2
+            f_bak.exists.should_be_false
+            f.delete_if_exists
+
+        Test.specify "should allow to appending bytes to a new file" <|
```

Comment:
Ok, here I'm extremely picky and it feels weird since it's not may native tongue, but I'm pretty sure it should either be `allow appending` or `allow to append` but not `allow to appending`.